Model/Player.py

```diff
@@ -0,0 +1,26 @@
+from Server.Database.DBMethods import DBMethods
+
+
+class Player:
+    def __init__(self, player_id, username, password):
+        self.id = player_id
+        self.username = username
+        self.password = password
+
+    def to_dict(self):
+        return self.__dict__
+
+    @staticmethod
+    def create_object_from_details_in_db(player_username):
+        player_as_dict = DBMethods.get_player_details_from_db(player_username)
+        if not player_as_dict:
+            return False
+        player_id = player_as_dict['id']
+        player_password = player_as_dict['password']
+        player_as_object = Player(player_id, player_username, player_password)
```

Comment:
While this function logic can be inside the DBMethods, then it is really a nice layer that converts the DB information to your real model objects :) 

because In case someone would like to change the player both of those functions require some change. 

For the clean and the fast search you can keep the create_object_from_details_in_db function that just call the DBMethods, then you keep the OOP concepts.